src/main/java/chess/controller/web/GameController.java

```diff
@@ -0,0 +1,66 @@
+package chess.controller.web;
+
+import chess.controller.dto.WinnerDto;
+import chess.domain.board.position.Position;
+import chess.domain.piece.Owner;
+import chess.service.GameService;
+import chess.service.RoomService;
+import chess.view.OutputView;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.sql.SQLException;
+import java.util.List;
+
+@Controller
+@RequestMapping("/game")
+public class GameController {
+    private final RoomService roomService;
+    private final GameService gameService;
+
+    public GameController(RoomService roomService, GameService gameService) {
+        this.roomService = roomService;
+        this.gameService = gameService;
+    }
+
+    @GetMapping("/load/{roomId}")
```

Comment:
```suggestion
    @GetMapping("/{roomId}")
```
같이 간결하게 표현하는 건 어떤가요 REST 방식에서 행위는 HTTP METHOD 를 통해서 표현하는것을 권장합니다.